freqtrade/optimize/hyperopt.py

```diff
@@ -78,6 +78,12 @@ def __init__(self, config: Dict[str, Any]) -> None:
             self.max_open_trades = 0
         self.position_stacking = self.config.get('position_stacking', False),
 
+        if self.has_space('sell'):
+            # Make sure experimental is enabled
+            if 'experimental' not in self.config:
+                self.config['experimental'] = {}
```

Comment:
Side note, probably for another time - this should be moved to `configuration.py`... probably right next to the initialization of `internals`...

Reply:
> Side note for another time - this should be moved to `configuration.py`... probably right next to the initialization of `internals`...

I also was thinking of it, right.